### ISSUE TITLE
SPA route still requires generateStaticParams

### ISSUE THREAD

**Issue by pandeiro** [onejs/one#236]
When running `npm run build:web`, the compilation fails for an app where `web.defaultRenderMode: 'spa'` and a dynamic route is using the default render mode, because `generateStaticParams` is missing:

```
 ERROR  [one] Error: Missing generateStaticParams                                                                               12:11:17 PM

  Route /[id] of type spa must export generateStaticParams so build can complete.

  See docs on generateStaticParams:
    https://onestack.dev/docs/routing-exports#generatestaticparams
```

For SPA this shouldn't be the case, right?

**Comment by pandeiro** [comment]
Including an empty stub function can be used as a workaround in the meantime:

```js
export async function generateStaticParams() {}
```

**Comment by natew** [comment]
the docs do show an array for me (maybe @zetavg fixed it), but thanks for the heads up. fixed in 1.1.344
